R/bambu-quantify.R

```diff
@@ -24,14 +24,16 @@ bambu.quantify <- function(readClass, annotations, emParameters,
     incompatibleCounts <- incompatibleCounts[data.table(GENEID = unique(mcols(annotations)$GENEID)), on = "GENEID"]
     incompatibleCounts[is.na(counts), counts := 0]
     setnames(incompatibleCounts, "counts", colnames(readClass))
+    compatibleCounts <- calculateCPM(compatibleCounts, incompatibleCounts)
```

Comment:
at this point in the code, incompatibleCounts$counts doesn't exist because of line 26. The column is labeled by the sample name so the sum($counts) is 0. This would change the calculation I think?
This means the sum of $CPM is 1000000 even though the existance of incompatibleCounts should make this lower

Reply:
thanks for pointing this out! fixed it now